string classes
uint16 idx
float32 x
float32 y
float32 z
float32 xMin
float32 yMin
float32 zMin
float32 xMax
float32 yMax
float32 zMax
raw4cam atYmin
raw4cam atYmax
raw4cam atZmin
raw4cam atZmax